classesExamples/PhysicianRobot.py

```diff
@@ -0,0 +1,14 @@
+from classesExamples.Robot import Robot
+
+
+class PhysicianRobot(Robot):
+    pass
+
+
+x = Robot("Marvin")
```

Comment:
I understand these names are part of the practice (`x`, `y`), however keep in mind that variable names should be expressive, naming is hard I know, but the better the naming the better you will be able to read your code later.